man/systemd.device.xml

```diff
@@ -152,9 +151,135 @@
   <refsect1>
     <title>Options</title>
 
-    <para>Device unit files may include [Unit] and [Install] sections, which are described in
-    <citerefentry><refentrytitle>systemd.unit</refentrytitle><manvolnum>5</manvolnum></citerefentry>. No
-    options specific to this file type are supported.</para>
+    <para>Device unit files may include [Device], [Unit] and [Install] sections. The [Device] section includes device specific properties. The [Unit] and [Install] sections are described in
+    <citerefentry><refentrytitle>systemd.unit</refentrytitle><manvolnum>5</manvolnum></citerefentry>.
+    </para>
+
+    <variablelist class='unit-directives'>
+      <varlistentry>
+        <term><varname>IOCostQOSCtrl=</varname></term>
+        <listitem><para>Takes the special options <option>auto</option> and <option>user</option>. <literal>io.cost.qos</literal> is disabled if unspecified.
```

Comment:
What if user configures just `IOCostQOSCtrl=`? I think the current implementation would trigger a warning, however, it should be a valid input if the user wants to disable the io.cost for the device (e.g. in a drop-in override).

Reply:
Without a warning?